ScoutSuite/providers/azure/configs/services.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, metadata=None, thread_config=4, **kwargs):
         self.storageaccounts = StorageAccountsConfig(thread_config=thread_config)
         self.monitor = MonitorConfig(thread_config=thread_config)
         self.sqldatabase = SQLDatabaseConfig()
-        self.securitycenter = SecurityCenterConfig(thread_config=thread_config)
+        self.securitycenter = SecurityCenterConfig()
```

Comment:
Weren't we getting rid of the `Config` suffix? I'm confused. 😵 I started removing them in my other PRs.

Reply:
My bad, I just realized it's an alias. Why use an alias though?